packages/gen-ai-hub/src/utils/deployment-resolver.test.ts

```diff
@@ -0,0 +1,133 @@
+import nock from 'nock';
+import {
+  mockClientCredentialsGrantCall,
+  mockAiCoreEnvVariable,
+  aiCoreDestination
+} from '../../../../test-util/mock-http.js';
+import { resolveDeployment } from './deployment-resolver.js';
+
+describe('Deployment resolver', () => {
+  beforeEach(() => {
+    mockAiCoreEnvVariable();
```

Comment:
[req] mocking AI core env variable is done in the global setup. SO you don't need to do it anywhere else.